src/MariaDbEventStore.php

```diff
@@ -806,4 +807,14 @@ private function createSchemaFor(string $tableName): void
             }
         }
     }
+
+    private function convertToColumn(array &$match): void
```

Comment:
Regarding MariaDbIndexedColumnStrategy
I'd just add a check here:

```php
if($this->persistenceStrategy instanceof MariaDbIndexedColumnStrategy) { ... }

Reply:
This way public API doesn't change but the performance bottleneck is solved internally. This should be documented, though. In the next major release we can respect the difference in the public API as well.